### PR TITLE
add toggle menu

### PR DESCRIPTION
**Pull request by franciscoPonceDev**
Add toggle menu to navbar.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,15 +27,16 @@
...
 <body>
-        <header class="nav-bar">
+        <header id="nav" class="nav-bar">
             <h3>Francisco Ponce</h3>
+            <img id="close" src="./assets/closing-btn.svg" alt="Close menu" class="closing-btn">
             <ul class="desktop-only">
                 <li><a class="link" href="#portfolio">Portfolio</a></li>
                 <li><a class="link" href="#about-me">About</a></li>
                 <li><a class="link" href="#contact">Contact</a></li>
                 <li><a class="link" href="#contact"><i class="far fa-envelope fa-lg"></i></a></li>
```
Comment (MrBrN197) [inline index.html]: - [x] Your design is well, Please notice though that you have an extra `li` tag in the hamburger menu; this is causing your menu to leave an extra border below the "Contact". You have set the `display` to `none` for the envelope which is good although you also need to handle the `border-bottom` on the `li` element itself. Please kindly fix this issue as this doesn't match the reference design.

Your Design | Reference Design
-|-
image|image

Comment (MrBrN197) [inline index.html]: - [x] You are currently missing the close button for the hamburger popup. If you look at the Figma design you'll notice that this is supposed to be preset. please implement this design.
- [x] Implement the functionality for closing the hamburger when you click on the close icon.

Your Design | Reference Design
-|-
image|image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @franciscoPonceDev  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Design matches well
- ✔ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @franciscoPonceDev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**